lib/extensions/descendant_loader.rb

```diff
@@ -0,0 +1,269 @@
+require 'ruby_parser'
```

Comment:
I'd rather keep the generic "use whichever version you think is best" spelling (and would really argue that it's on ruby_parser to not load everything just in case)...

But I can actually do even better -- I've moved the require to only occur if we do a parse.